content/docs/guides/10-minute-tutorial.md

```diff
@@ -1234,36 +1241,41 @@ When we run this test, it will fail.
 
 {{% block "java" %}}
 ```shell
+-------------------------------------------------------
```

Comment:
This was missing in previous outputs. :thinking: 
Added here to be consistent with the other console output snippets. :muscle: